suite-native/atoms/src/Sheet/useBottomSheetAnimation.ts

```diff
@@ -10,15 +10,16 @@ import {
 } from 'react-native-reanimated';
 import { useCallback, useEffect } from 'react';
 import { PanGestureHandlerGestureEvent } from 'react-native-gesture-handler';
-import { Dimensions, NativeScrollEvent } from 'react-native';
+import { NativeScrollEvent } from 'react-native';
 
 import { useNativeStyles } from '@trezor/styles';
+import { getScreenHeight } from '@trezor/env-utils';
 
 type GestureHandlerContext = {
     translatePanY: number;
 };
 
-const SCREEN_HEIGHT = Dimensions.get('screen').height;
+const SCREEN_HEIGHT = getScreenHeight();
```

Comment:
side note: We don't have to strictly use this env functions everywhere, it probably makes sense only in suite-common folder, in suite-native this should be optional.